client/my-sites/media-library/list-no-content.jsx

```diff
@@ -10,57 +11,65 @@ import EmptyContent from 'components/empty-content';
 import UploadButton from './upload-button';
 import { userCan } from 'lib/site/utils';
 
-export default React.createClass( {
-	displayName: 'MediaLibraryListNoContent',
-
-	propTypes: {
+class MediaLibraryListNoContent extends Component {
+	static propTypes = {
 		site: React.PropTypes.object,
 		filter: React.PropTypes.string,
 		source: React.PropTypes.string,
-	},
+	};
 
 	getLabel() {
-		switch ( this.props.filter ) {
+		const {
+			filter,
+			translate,
+		} = this.props;
+
+		switch ( filter ) {
+			case 'this-post':
+				return translate( 'There are no media uploaded to this post.', {
```

Comment:
Yes. Adding context is only recommended when you have a definitive requirement to firmly differentiate between ambiguous variants of a string. Usually the strings will be something short like "Post" (eg, the noun v the verb).  A comment, however, is quite welcome!

Aside: Is it a conscientious choice to treat "media" that way in the copy or does "There isn't any media uploaded to the post." make sense?